wayfire.ini

```diff
@@ -73,11 +76,14 @@ plugins = \
 # https://github.com/WayfireWM/wayfire/wiki/Configuration#blur
 
 # Close focused window.
-close_top_view = <super> KEY_W | <super> BTN_MIDDLE | <alt> KEY_F4
+close_top_view = <super> KEY_W | <super> BTN_MIDDLE | <alt> KEY_FN_F4 | <alt> KEY_F4
 
 # Workspaces arranged into a grid: 3 × 3.
 vwidth = 3
 vheight = 3
+# Enable Xwayland
+# also install xorg-xhost to run xwaland apps with elevated privilage
+xwayland = true
```

Comment:
+1, you can link to the Tips & Tricks wiki page instead of explaining xorg-xhost, since we already have information for xwayland apps and sudo there.